data/3e/dragonwing_games/bastion_press/faeries/faeries_races.lst

```diff
@@ -6,14 +6,14 @@ SOURCELONG:Faeries	SOURCESHORT:Faeries	SOURCEWEB:http://dragonwing.net/Faeries.h
-Half-Fey	FAVCLASS:HIGHESTLEVELCLASS	STARTFEATS:1	SIZE:M	MOVE:Walk,30		FACE:5,5	REACH:5	VISION:Low-Light Vision		LANGBONUS:ALL				BONUS:CASTERLEVEL|Half-Fey|TL	BONUS:DC|CLASS.Half-Fey|CHA																						AUTO:LANG|Fae|Fey|Sylvan|Common											ABILITY:Special Ability|AUTOMATIC|Fey Blood|Fey Racial - Half-Fey|Halt Aging ~ Half-Fey								RACETYPE:Fey							LEVELADJUSTMENT:1				SOURCEPAGE:p.10	SPELLS:Innate|TIMES=3|CASTERLEVEL=max(TL,1)|Good Luck
-Scath		FAVCLASS:HIGHESTLEVELCLASS	STARTFEATS:1	SIZE:M	MOVE:Walk,30		FACE:5,5	REACH:5	VISION:Darkvision (60')		LANGBONUS:Aquan,Common,Terran		BONUS:CASTERLEVEL|Scath|TL	BONUS:DC|CLASS.Scath|CHA	BONUS:STAT|WIS|2	BONUS:STAT|DEX|1	BONUS:STAT|CON|-1	BONUS:STAT|CHA|-2										AUTO:LANG|Fae|Fey|Sylvan|Undercommon										ABILITY:Special Ability|AUTOMATIC|Fey Blood|Fey Racial - Scath|Light Sensitivity									RACETYPE:Fey													SOURCEPAGE:p.11	SPELLS:Innate|TIMES=1|CASTERLEVEL=max(TL,1)|Dispel Magic,13+CHA	SPELLS:Innate|TIMES=3|CASTERLEVEL=max(TL,1)|Deeper Darkness,13+CHA
-Sprite	FAVCLASS:Cleric			STARTFEATS:1	SIZE:S	MOVE:Walk,20,Fly,40	FACE:5,5	REACH:5	VISION:Low-Light Vision		LANGBONUS:Auran,Celestial,Elven	BONUS:CASTERLEVEL|Sprite|TL	BONUS:DC|CLASS.Sprite|CHA	BONUS:STAT|DEX|2	BONUS:STAT|CHA|1	BONUS:STAT|INT|-1	BONUS:STAT|STR|-2	BONUS:VAR|RacialTraitsECL|(TL+9)/8|PRELEVEL:MIN=3	AUTO:LANG|Fae|Fey|Sylvan|Common											ABILITY:Special Ability|AUTOMATIC|Fey Blood|Fey Racial - Sprite|Halt Aging			DEFINE:RacialTraitsECL|0			RACETYPE:Fey	TEMPLATE:Sprite Racial Traits		LEVELADJUSTMENT:1				SOURCEPAGE:p.12	SPELLS:Innate|TIMES=1|CASTERLEVEL=max(TL,1)|Charm Person,11+CHA	SPELLS:Innate|TIMES=3|CASTERLEVEL=max(TL,1)|Good Luck|Invisibility,12+CHA
+# Race Name	Favored Class			Starting Feats	Size		Move				Face/Space	Reach		Vision				Bonus Languages				Caster level			Bonus DC				Stat bonus											Modify VAR								Auto Language					Auto Weapon Prof							Ability														Define				Spell Res.	Main Race Type	Template					Level Adjustment				Challenge Rating		Source Page		Innate Spells
+Bogie		FAVCLASS:Fighter			STARTFEATS:1	SIZE:S	MOVE:Walk,20		FACE:5,5	REACH:5	VISION:Darkvision (90')		LANGBONUS:Common,Goblin,Orc		BONUS:CASTERLEVEL|Bogie|TL	BONUS:DC|CLASS.Bogie|CHA	BONUS:STAT|STR|2	BONUS:STAT|INT|1	BONUS:STAT|CHA|-1	BONUS:STAT|DEX|-2	BONUS:VAR|RacialTraitsECL|(TL+9)/8|PRELEVEL:MIN=3	AUTO:LANG|Fae|Fey|Shadow												ABILITY:Special Ability|AUTOMATIC|Does Not Age|Fey Blood|Fey Racial - Bogie			DEFINE:RacialTraitsECL|0			RACETYPE:Fey	TEMPLATE:Bogie Racial Traits		LEVELADJUSTMENT:RacialTraitsECL+1	CR:RacialTraitsECL+1	SOURCEPAGE:p.4	SPELLS:Innate|TIMES=1|CASTERLEVEL=max(TL,1)|Charm Person,11+CHA	SPELLS:Innate|TIMES=3|CASTERLEVEL=max(TL,1)|Good Luck|Invisibility,12+CHA
```

Comment:
Does CR take a Variable?

Reply:
I asked on the slack, got a positive answer. I'm not sure what to do with it otherwise.